### PR TITLE
Hero beta fixes merge

### PR DESCRIPTION
**Pull request by ryanzhoudev**
Remember to add %es% and %elemental_special% to website faq on merge

### CODE REVIEW

**Comment by kristofbolyai** [comment]
Description does not mention this, but DO NOT MERGE until 2.0 release. This PR is open for code reviews.

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 	id "com.github.johnrengelman.shadow" version "7.1.0"
 }
 
-def versionObj = new Version(major: 1, minor: 11, revision: 3)
+def versionObj = new Version(major: 1, minor: 11, revision: 4)
```
Comment (kristofbolyai) [inline build.gradle]: What is this version bump? I thought we only do these on releases.

Comment (ryanzhoudev) [reply build.gradle]: It appears that the merge from development into hero-beta (557, 556) is showing up in changed files, even though it is not changed

Comment (ryanzhoudev) [reply build.gradle]: We are currently on 1.11.4 in CE

Comment (kristofbolyai) [reply build.gradle]: uh..

**Review by DevScyu — DISMISSED** [review]

File: src/main/java/com/wynntils/modules/core/instances/TotemTracker.java [hunk 2]
```diff
@@ -151,11 +151,11 @@ public void onTotemSpawn(PacketEvent<SPacketSpawnObject> e) {
     }
 
     public void onTotemSpellCast(SpellEvent.Cast e) {
-        if (e.getSpell().equals("Totem") || e.getSpell().equals("Sky Emblem")) {
+        if (e.getSpell().equals("Totem")) {
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/core/instances/TotemTracker.java]:
```suggestion
        if (SpellType.TOTEM.getName().equals(e.getSpell())) {
```
I think we should use the already set name in the SpellType enum to reduce the chance of spelling mistakes and only have one place to fix them in the future.

File: src/main/java/com/wynntils/modules/core/instances/TotemTracker.java [hunk 3]
```diff
@@ -151,11 +151,11 @@ public void onTotemSpawn(PacketEvent<SPacketSpawnObject> e) {
...
             totemCastTimestamp = System.currentTimeMillis();
-            heldWeaponSlot =  McIf.player().inventory.currentItem;
+            heldWeaponSlot = McIf.player().inventory.currentItem;
             checkTotemSummoned();
-        } else if (e.getSpell().equals("Uproot") || e.getSpell().equals("Gale Funnel")) {
+        } else if (e.getSpell().equals("Uproot")) {
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/core/instances/TotemTracker.java]:
```suggestion
        } else if (SpellType.UPROOT.getName().equals(e.getSpell())) {
```

**Review by magicus — DISMISSED** [review]
I assume this is OK. I've scanned through the changes and found nothing I reacted to. But I have not played 2.0 beta much and can't say anything about how well the PR responds to actual changes in Wynncraft. But I trust thatsince  this has been build up by a series of incremental patches, it should be good.

**Review by HighCrit — DISMISSED** [review]

**Review by HighCrit — APPROVED** [review]

**Review by HeyZeer0 — APPROVED** [review]

**Pull request merged**